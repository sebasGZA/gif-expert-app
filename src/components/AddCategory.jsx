import { useState } from "react";

export const AddCategory = () => {
  const [inputValue, setInputValue] = useState("One Punch");

  const onInputChange = ({ target }) => {
    setInputValue(target.value);
  };

  const onSubmit = (event) => {
    event.preventDefault();
    console.log(inputValue);
  };

  return (
    <form onSubmit={onSubmit}>
      <input
        type="text"
        placeholder="Search gifs"
        value={inputValue}
        onChange={onInputChange}
      />
    </form>
  );
};
