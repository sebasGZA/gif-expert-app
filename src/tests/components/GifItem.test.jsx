import { render } from "@testing-library/react";
import { GifItem } from "../../components/GifItem";

describe("Tests in <GifItem/>", () => {
  const title = "Dragon Ball";
  const url = "https://dragon-ball.com/goku.jpg";

  test("Should make match with snapshot", () => {
    const { container } = render(<GifItem title={title} url={url} />);
    expect(container).toMatchSnapshot();
  });
});
