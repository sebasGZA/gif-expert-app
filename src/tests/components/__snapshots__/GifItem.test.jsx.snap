// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <GifItem/> Should make match with snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Dragon Ball"
      src="https://dragon-ball.com/goku.jpg"
    />
    <p>
      Dragon Ball
    </p>
  </div>
</div>
`;
